deps/chakrashim/lib/chakra_shim.js

```diff
@@ -89,12 +120,34 @@
   // Parse 'stack' string into StackTrace frames. Skip top 'skipDepth' frames,
   // and optionally skip top to 'startName' function frames.
   function parseStack(stack, skipDepth, startName) {
+    var curr = parseStack;
+
     var splittedStack = stack.split('\n');
```

Comment:
Sure, updated the code to use regex here as well as where we extract line/column numbers.